tasks.yaml

```diff
@@ -59,3 +59,11 @@ tasks:
           openssl req -new -key test.pem -out test.csr -config csr.conf -subj "/C=US/ST=Colorado/L=Colorado Springs/O=Defense Unicorns/CN=uds.dev"
           openssl x509 -req -in test.csr -CA ca.crt -CAkey ca.pem -CAcreateserial -out test.cer -days 365 -extensions v3_ext -extfile csr.conf
           zip src/authorized_certs ca.crt ca.pem test.cer test.csr test.pem
+
+  - name: uds-core-integration-test
+    description: "UDS Core integration test"
+    actions:
+      - cmd: uds deploy ghcr.io/defenseunicorns/packages/uds/bundles/k3d-core-istio-dev:0.16.1 --confirm
+      - task: dev-build
+      - cmd: git clone https://github.com/defenseunicorns/uds-core.git
```

Comment:
Probably want to pull a specific version of uds-core here; should be consistent with the base bundle version.

Reply:
@rjferguson21 add the version to be pulled as a renovate tracked variable. also was added in the cypress integration PR changes